app/models/watched_encode.rb

```diff
@@ -36,6 +36,12 @@ class WatchedEncode < ActiveEncode::Base
       encode.output.collect! do |output|
         if output.format == "vtt"
           new_file = SupplementalFile.new(tags: ['caption'], parent_id: record.master_file_id)
+          new_file.label = output.label.presence
+          begin
+            new_file.language = LanguageTerm.find(output.language).code if output.language.present?
+          rescue LanguageTerm::LookupError
+            new_file.language = nil
+          end
```

Comment:
Not a big deal but I've read that begin/rescue has an overhead cost so it would be better here to skip that block by moving the guard clause outside of it.
```suggestion
          if output.language.present?
            begin
              new_file.language = LanguageTerm.find(output.language).code
            rescue LanguageTerm::LookupError
              new_file.language = nil
            end
          end
```

Reply:
That makes sense. Good catch!